debug-container/Dockerfile

```
@@ -1,36 +1,57 @@
 # Locking this to a specific Fedora version for now. New releases of Postgresql, and Fedora versions going out of
 # support will drive this upgrade every 6-12 months.
-FROM quay.io/fedora/fedora:43@sha256:5f32b834046ca567cf7956d8bdf680c3f87c9fa376421b4cf28cf5d023ee0d86
+FROM quay.io/fedora/fedora:43.20250718.0
```

Please change it to the last 43 tag:

```suggestion
FROM quay.io/fedora/fedora:43@sha256:f5a7f589fe0a95baa5a09766637192e5703e51b00c60183fc42f8461578d3d64)
```